src/main/java/vct/col/rewrite/Triggers.scala

```diff
@@ -0,0 +1,155 @@
+package vct.col.rewrite
+
+import vct.col.ast.`type`.ASTReserved
+import vct.col.ast.expr.{Binder, BindingExpression, Dereference, MethodInvokation, NameExpression, NameExpressionKind, OperatorExpression}
+import vct.col.ast.generic.ASTNode
+import vct.col.ast.stmt.decl.{ASTSpecial, DeclarationStatement, ProgramUnit}
+import vct.col.ast.util.AbstractRewriter
+import vct.col.ast.expr.StandardOperator.{EQ, GT, GTE, Implies, LT, LTE, Member, NEQ, Scale, Size, Subscript}
+import vct.col.ast.expr.constant.{ConstantExpression, StructValue}
+
+case class UnrecognizedExpression(node: ASTNode) extends Exception
+
+case class Triggers(override val source: ProgramUnit) extends AbstractRewriter(source) {
```

Comment:
I think I generally understand how this class works (find all subexpressions that are more than just a variable but not a permission or binder, i.e. f(x), if I'm correct?), but maybe a short documentation at the top of the class might still be good.